base/commandLine.cpp

```diff
@@ -68,6 +69,13 @@ static const char HELP_STRING[] =
 	"  -z, --list-games         Display list of supported games and exit\n"
 	"  -t, --list-targets       Display list of configured targets and exit\n"
 	"  --list-saves=TARGET      Display a list of saved games for the game (TARGET) specified\n"
+	"  -a, --add                Add a game from current or specified directory\n"
+	"                           Use --path=PATH before -a, --add to specify a directory.\n"
+	"  --massadd                Add all games from current or specified directory and all sub directories\n"
```

Comment:
We've discussed the issue a bit with @criezy, see:

TL;DR, I don't find this very intuitive and I don't especially like the idea of interaction (if you want interaction, you'd use a GUI after all...)

The semantics I'd like to propose is:

```
-p <dir> --add                adds all games
-p <dir> --detect             enumerates dectected games in <dir>
-p <dir> --game <id> --add     adds just game <id>
```

How do you like it?